config/management/operational/src/governance.rs

```diff
@@ -0,0 +1,87 @@
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct AddValidator {
+    #[structopt(long, help = "JSON-RPC Endpoint (e.g. http://localhost:8080")]
+    host: String,
+    #[structopt(long)]
+    chain_id: ChainId,
+    #[structopt(long, help = "The validator address to be added")]
+    address_to_add: AccountAddress,
+    #[structopt(flatten)]
+    libra_root: ValidatorBackend,
+}
+
+impl AddValidator {
+    pub fn execute(self) -> Result<TransactionContext, Error> {
+        let client = JsonRpcClientWrapper::new(self.host);
+        if let Err(e) = client.validator_config(self.address_to_add) {
+            return Err(Error::JsonRpcReadError("validator-config", e.to_string()));
+        }
+        let seq_num = client.sequence_number(libra_root_address())?;
+        let txn = RawTransaction::new_script(
+            libra_root_address(),
+            seq_num,
+            transaction_builder::encode_add_validator_script(self.address_to_add),
+            constants::MAX_GAS_AMOUNT,
+            constants::GAS_UNIT_PRICE,
+            constants::GAS_CURRENCY_CODE.to_owned(),
+            RealTimeService::new().now() + constants::TXN_EXPIRATION_SECS,
+            self.chain_id,
+        );
+        let mut storage =
+            StorageWrapper::new(self.libra_root.name(), &self.libra_root.validator_backend)?;
+        let signed_txn = storage.sign(LIBRA_ROOT_KEY, "add-validator", txn)?;
+        client.submit_transaction(signed_txn)
+    }
+}
+
+#[derive(Debug, StructOpt)]
+pub struct RemoveValidator {
+    #[structopt(long, help = "JSON-RPC Endpoint (e.g. http://localhost:8080")]
+    host: String,
+    #[structopt(long)]
+    chain_id: ChainId,
+    #[structopt(long, help = "The validator address to be removed")]
+    address_to_remove: AccountAddress,
+    #[structopt(flatten)]
+    libra_root: ValidatorBackend,
+}
+
+impl RemoveValidator {
+    pub fn execute(self) -> Result<TransactionContext, Error> {
+        let client = JsonRpcClientWrapper::new(self.host);
+        if let Err(e) = client.validator_info(self.address_to_remove) {
+            return Err(Error::JsonRpcReadError("validator-info", e.to_string()));
+        }
+        let seq_num = client.sequence_number(libra_root_address())?;
+        let txn = RawTransaction::new_script(
+            libra_root_address(),
+            seq_num,
+            transaction_builder::encode_remove_validator_script(self.address_to_remove),
+            constants::MAX_GAS_AMOUNT,
+            constants::GAS_UNIT_PRICE,
+            constants::GAS_CURRENCY_CODE.to_owned(),
+            RealTimeService::new().now() + constants::TXN_EXPIRATION_SECS,
+            self.chain_id,
+        );
+        let mut storage =
+            StorageWrapper::new(self.libra_root.name(), &self.libra_root.validator_backend)?;
+        let signed_txn = storage.sign(LIBRA_ROOT_KEY, "remove-validator", txn)?;
+        client.submit_transaction(signed_txn)
+    }
```

Comment:
Nit: Not for you necessarily, but in my next refactor I want to refactor this out, as the transaction, signing, and submission are all the same, except the `script` and the `signing name`.  It's also basically the same as in validator config:

https://github.com/libra/libra/blob/master/config/management/src/validator_config.rs#L97-L111